web/src/auth/SignupPage.js

```diff
@@ -198,9 +199,20 @@ class SignupPage extends React.Component {
   onFinish(values) {
     const application = this.getApplicationObj();
 
+    const customFields = {};
+    const customFieldKeys = this.form.current.getFieldsValue(true);
+    for (const [key, value] of Object.entries(customFieldKeys)) {
+      if (key.startsWith("custom_field_")) {
```

Comment:
we should not allow user to enter string include space or special character , and we should check customField in backend to prevent user add external field

Reply:
OK, I have also seen the previous review and I will make changes later.

Reply:
`we should not allow user to enter string include space or special character` this means we we should limit user to input special character to the label field of signup table